core/tests/protractor_desktop/navigation.js

```diff
@@ -220,6 +220,13 @@ describe('Static Pages Tour', function() {
       by.css('.protractor-test-thanks-page')).isPresent()).toBe(true);
   });
 
+  it('should visit the Volunteer page', async function() {
+    await browser.get('/volunteer');
+    await waitFor.visibilityOf(
```

Comment:
Why did we remove the `await waitFor.pageToFullyLoad()`?

Reply:
Fixed.